app/controllers/sms_controller.rb

```diff
@@ -44,7 +44,9 @@ def process_failure(result)
       result.to_h
     )
 
-    head :forbidden
+    # Invalid requests don't require a response for now, and 4xx
+    # errors from random incoming messages junk up the logs
+    head :ok
```

Comment:
I think we may want to render a 403 here under a lot of circumstances, if for example the signature on the request is incorrect or our app thinks the request is malformed. Otherwise those errors won't be reported to us.